tests/vmi_multus_test.go

```diff
@@ -580,6 +580,12 @@ var _ = Describe("SRIOV", func() {
 
 	tests.BeforeAll(func() {
 		tests.BeforeTestCleanup()
+		// Check if the hardware supports SRIOV
+		sriovcheck := checkSriovEnabled(virtClient, sriovResourceName)
+		if !sriovcheck {
+			Fail("Sriov is not enabled in this environment. Skip these tests using - export FUNC_TEST_ARGS='--ginkgo.skip=SRIOV'")
```

Comment:
Please change this Fail() to a Skip().

Because checkSriovEnabled can't check if SR-IOV is *supposed* to be enabled, this ends up being way too aggressive. The only way a Fail() is acceptable here is if the user explicitly asks for SR-IOV tests to be executed. We don't currently have a framework for such a thing.